linux/user-and-file-management/bash-environment/manipulating-environment-variables-iv.md

```diff
@@ -0,0 +1,57 @@
+---
+author: jfarmer
+
+levels:
+
+  - basic
+
+  - beginner
+
+aspects:
+  - introduction
+  - workout
+
+type: normal
+
+category: must-know
+
+inAlgoPool: false
+
+tags:
+
+  - introduction
+
+
+---
+
+# Manipulating Environment Variables (Part 4)

```

Comment:
headline should be "Variable creation syntax", and remove the first heading from the content section